src/components/AreaManagementFilter/AreaManagementFilter.jsx

```diff
@@ -0,0 +1,159 @@
+import { React, useEffect, useState } from 'react';
```

Comment:
It looks like if I add too many areas, the states go outside the modal window

![image](https://user-images.githubusercontent.com/57849152/166341083-d8db238d-dbfe-469f-852d-fb7623007550.png)